pagopa/io-app
# chore: [IOPID-1186] Add bottomsheet logic for security suggestion acknowledged

## Short description
The security advice bottomsheet has been modified. the bottomsheet is displayed in the opt-in step, if it is not displayed, the bottomsheet is shown to the user after the onboarding/first onboarding flow. The bottomsheet must be shown both in the case of fast login flow or classic login flow. 
[This is the figma](https://www.figma.com/file/BEXvjQmBkMRRFiNcMgE8g3/Login-Veloce?node-id=2885%3A19598&mode=dev)


## List of changes proposed in this pull request
- Change text of bottomsheet 
- Change text of NewOptInScreen.tsx
- Componentize bottomsheet 
- Add button to bottomsheet after login
- Add value into persistent store to maintain the information about the security suggestion acknowledged
- Fix go back logic on change email screen

https://github.com/pagopa/io-app/assets/83651704/a0dda488-317e-49aa-979b-6e05a371c819

## How to test
First of all if you need to test the first onboarding flow, into the dev-server must have `firstOnboarding: true`. 
In the app, the .env file must have `FAST_LOGIN_ENABLED=YES` and `FAST_LOGIN_OPTIN=YES`. Once you have changed all this data, run `yarn postinstall` on the app and then `yarn run-ios` (or yarn run-android) (NB: the app must be installed from new)
The test cases are the same as you see in the videos above
